src/models/index.js

```diff
@@ -18,9 +18,8 @@ if (env !== 'production') {
   sequelize = new Sequelize(process.env.DATABASE_URL);
 }
 
-fs
-  .readdirSync(__dirname)
-  .filter((file) => (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js'))
+fs.readdirSync(__dirname)
+  .filter((file) => file.indexOf('.') !== 0 && file !== basename && file.slice(-3) === '.js')
```

Comment:
Remove the parenthesis around  the `file` argument on line 22 so that hound can make us rest

Reply:
:-)

Reply:
I've resolved it. Thanks.

Reply:
I cant see it being implemented. Take a look at line 22

Reply:
Okay, I'm sorry.
I've resolved it now. Thanks.